lib/models/Order.js

```diff
@@ -11,6 +13,11 @@ module.exports = class Order {
 
   static async insert({ product, quantity }) {
     // TODO: Implement me
+    const { rows } = await pool.query(
+      'INSERT INTO orders(product, quantity) VALUES ($1, $2) RETURNING *;',
+      [product, quantity]
+    );
+    return new Order(rows[0]);
```

Comment:
We could add `if(!rows[0])` here instead of instead of inside of our catch in the orders controller